pkg/util/spd.go

```diff
@@ -312,6 +312,27 @@ func InsertSPDBusinessIndicatorStatus(status *apiworkload.ServiceProfileDescript
 	status.BusinessStatus = append(status.BusinessStatus, *serviceBusinessIndicatorStatus)
 }
 
+func InsertSPDAggMetrics(status *apiworkload.ServiceProfileDescriptorStatus,
+	serviceAggPodMetrics *apiworkload.AggPodMetrics,
+) {
+	if status == nil || serviceAggPodMetrics == nil {
+		return
+	}
+
+	if status.AggMetrics == nil {
+		status.AggMetrics = []apiworkload.AggPodMetrics{}
+	}
+
+	for i := range status.AggMetrics {
+		if status.AggMetrics[i].Scope == serviceAggPodMetrics.Scope {
```

Comment:
it needs add a method like `GetSupportedExtendedIndicatorSpec` in IndicatorPlugin add some logic in `mergeIndicatorStatus`  to make the scope is uniq.